_tools/check_docs.ts

```diff
@@ -59,6 +59,11 @@ const ENTRY_POINTS = [
   "../semver/mod.ts",
   "../streams/mod.ts",
   "../text/mod.ts",
+  "../testing/bdd.ts",
+  "../testing/mock.ts",
+  "../testing/snapshot.ts",
+  "../testing/time.ts",
+  "../testing/types.ts",
```

Comment:
`../testing/asserts.ts` is excluded because they're going to be removed at 1.0.0-rc.1